tests/testthat/test-crunch-data-frame-merge.R

```diff
@@ -0,0 +1,168 @@
+context("CrunchDataFrame merging")
```

Comment:
Are these tests just moved from the other file or are there changes? Hard to tell from the diff.